gen/calc.py

```diff
@@ -222,6 +222,20 @@ def validate_oauth_enabled(oauth_enabled):
     validate_true_false(oauth_enabled)
 
 
+def validate_network_default_name(dcos_overlay_network_default_name, dcos_overlay_network):
+    try:
+        overlay_network = json.loads(dcos_overlay_network)
+    except ValueError:
+        # TODO(cmaloney): This is not the right form to do this
+        assert False, "Provided input was not valid JSON: {}".format(dcos_overlay_network)
+
+    overlay_names = map(lambda overlay: overlay['name'], overlay_network['overlays'])
+
+    if dcos_overlay_network_default_name not in overlay_names:
+        assert False, "Default overlay network name does not reference a defined overlay network: {}".format(
```

Comment:
Please just raise an AssertionError with your message instead of doing ```assert False, my_message```

Better yet:
```
assert dcos_over_network_default_name in overlay_names, my_message
```